src/components/RadioButton/RadioButton.spec.jsx

```diff
@@ -0,0 +1,112 @@
+import _ from 'lodash';
+import assert from 'assert';
+import React from 'react';
+import SyntheticEvent from 'react/lib/SyntheticEvent';
+import sinon from 'sinon';
+import { mount, shallow } from 'enzyme';
+
+import describeWithDOM from '../../util/describe-with-dom';
+import { common } from '../../util/generic-tests';
+
+import RadioButton from './RadioButton';
+
+describe('RadioButton', () => {
+	common(RadioButton);
+
+	const booleanValues = [true, false];
+
+	describe('props', () => {
+		describe('isDisabled', () => {
+			it('sets the `disabled` attribute of the native radio button element.', () => {
+				_.forEach(booleanValues, (testValue) => {
+					const wrapper = shallow(<RadioButton isDisabled={testValue} />);
+
+					assert.equal(wrapper.find('input[type="radio"]').prop('disabled'), testValue);
+				});
+			});
+
+			it('defaults to `false`.', () => {
+				const wrapper = mount(<RadioButton />);
+
+				assert.equal(wrapper.prop('isDisabled'), false);
+			});
+		});
+
+		describe('isSelected', () => {
+			it('sets the `checked` attribute of the native radio button element.', () => {
+				_.forEach(booleanValues, (testValue) => {
+					const wrapper = shallow(<RadioButton isSelected={testValue} />);
+
+					assert.equal(wrapper.find('input[type="radio"]').prop('checked'), testValue);
+				});
+			});
+
+			it('defaults to `false`.', () => {
+				const wrapper = mount(<RadioButton />);
+
+				assert.equal(wrapper.prop('isSelected'), false);
+			});
+		});
+
+		describe('onSelect', () => {
+			it('defaults to the Lodash `noop` method.', () => {
+				const wrapper = mount(<RadioButton />);
+
+				assert.equal(wrapper.prop('onSelect'), _.noop);
+			});
+		});
+
+		describe('pass throughs', () => {
+			it('passes through all props not defined in `propTypes` to the native input.', () => {
```

Comment:
I was wondering if we could move this kind of test into a common test for "control" type components.